### PR TITLE
Add mark-down to notifications drawer

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-29370

### Before
before

### After
after

### CODE REVIEW

File: src/components/NotificationsDrawer/NotificationItem.tsx [hunk 1]
```diff
@@ -83,7 +85,9 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ notification, onNav
             <Label variant="outline" isCompact className="pf-u-mb-md">
               {notification.source}
             </Label>
-            <span className="pf-u-display-block">{notification.description}</span>
+            <span className="pf-u-display-block">
+              <ReactMarkdown remarkPlugins={[remarkGfm]}>{notification.description}</ReactMarkdown>
```
Comment (Hyperkid123) [inline src/components/NotificationsDrawer/NotificationItem.tsx:89]: Is the content always a markdown syntax?

Comment (karelhala) [reply src/components/NotificationsDrawer/NotificationItem.tsx:89]: Yes, notifications team is sending markdown for all notifications.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
The library does not have CJS output
```
 /home/runner/work/insights-chrome/insights-chrome/node_modules/react-markdown/index.js:9
```

You will have to transofrm it in jest env

**Review by aneelac22 — APPROVED** [review]

**Pull request merged**